client/state/data-layer/wpcom/sites/plugins/recommended/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { normalizePluginsList } from 'lib/plugins/utils';
+import { PLUGINS_RECOMMENDED_REQUEST } from 'state/action-types';
+import {
+	dispatchRecommendPluginsRequestFailure,
+	receiveRecommendedPlugins,
+} from 'state/plugins/recommended/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+
+export const fetch = action => {
+	const { siteId, limit = 6 } = action;
+
+	return [
```

Comment:
Seems like we no longer need this to be an array, and we can just return the action creator.